lib/artist.rb

```diff
@@ -1,19 +1,16 @@
-class Artist
-  extend Concerns::Findable
-  extend Concerns::BaseFunctions
-  include Concerns::MusicStoreActions
+class Artist < Base
 
   attr_accessor :name
   attr_reader :songs
-  @@all = []
```

Comment:
Have an instance all variable and convert it into a class variable using the class attribute accessor